common/tools/dev-tool/src/commands/test-proxy/stop.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { leafCommand, makeCommandInfo } from "../../framework/command";
+import { config } from "dotenv";
```

Comment:
why do you need dotenv here?

Reply:
Oops, just copied this from `start.ts` I think. Shouldn't be necessary.